install_deps.sh

```diff
@@ -262,5 +264,30 @@ git clone -b v1.37.1 --depth 1 --recurse-submodules https://github.com/grpc/grpc
     cd ${HOME} && \
     rm -r ${HOME}/grpc
 
-# After installing all libraries, let's make sure that they will be found at runtime
-ldconfig
+# libcurl needed for Prometheus-cpp
+cd ${HOME}
+wget ${WGET_FLAGS} --no-check-certificate https://curl.se/download/curl-7.66.0.tar.xz && \
```

Comment:
Isn't `--no-check-certificate` unsafe? We don't seem to need it elsewhere in the `install_deps.sh` script when using wget...

Reply:
Ha. The certificate can be validated via `curl` :D I will change this line.